initializr-service/src/main/resources/application.yml

```diff
@@ -104,6 +104,16 @@ initializr:
   dependencies:
     - name: Core
       content:
+        - name: Spring Shell
+          groupId: org.springframework.shell
+          artifactId: spring-shell-starter
+          id: spring-shell
+          description: Build shell-based clients
+          version: 2.0.0.M2
```

Comment:
It's a milestone so you may want to add a repositories section, e.g. `repository: spring-milestones`

Reply:
i've updated PR to reflect this. thanks for the good feedback.